src/besselj.jl

```diff
@@ -220,3 +220,16 @@ function log_besselj_small_arguments_orders(v, x::T) where T
     logout = -loggamma(v + 1) + v * log(x/2) + log(out)
     return logout
 end
+
+# valid when x < v (uniform asymptotic expansions)
+function besselj_debye(v, x)
+    T = eltype(x)
+    S = promote_type(T, Float64)
+    x = S(x)
+    n = v*log(v/x + sqrt((v/x)^2 - 1)) - sqrt(v^2 - x^2)
+    coef = inv(sqrt(2*T(pi))) * exp(-n) / (v^2 - x^2)^(1/4)
+    p = v / sqrt(v^2 - x^2)
+    p2  = v^2/fma(max(v,x), max(v,x), -min(v,x)^2)
```

Comment:
if `v<x`, isn't this just `v^2/(x^2 -v^2)`

Reply:
Yes - but `x<v` so `v^2/fma(v,v, -x^2)`. Fixed

Reply:
You should make `fma(v,v, -x^2)` a variable and use it since it will be cleaner and guarantee that you don't compute the same thing multiple times.